File: Python/bus.py
from landway import Landway

class Bus(Landway):

    def __init__(self, vehicle_id, vehicle_name, engine_type, capacity):
        super().__init__(vehicle_id, vehicle_name, engine_type)
        self.capacity = capacity
        self.passengers = 0

    def set_passengers(self, count):
        self.passengers = count

    def get_passengers(self):
        return self.passengers

    def get_capacity(self):
        return self.capacity

    def display_details(self):
        super().display_details()
        print("Bus Capacity    :", self.capacity)
        print("Passengers      :", self.passengers)

    def start_journey(self):
        print("Bus Journey Started Successfully...")

File: Java/Bus.java
class Bus extends Landway
{
    private int capacity;
    private int passengers;

    Bus(int id, String name, String engineType, int capacity)
    {
        super(id, name, engineType);
        this.capacity = capacity;
    }

    public void setPassengers(int passengers)
    {
        this.passengers = passengers;
    }

    @Override
    public void displayDetails()
    {
        super.displayDetails();
        System.out.println("Capacity        : " + capacity);
        System.out.println("Passengers      : " + passengers);
        System.out.println("Available Seats : " + (capacity - passengers));
    }

    @Override
    void startJourney()
    {
        System.out.println("Bus Journey Started Successfully...");
    }
}